src/ProjectTemplates/test/Templates.Blazor.WebAssembly.Tests/Templates.Blazor.WebAssembly.Tests.csproj

```
@@ -6,6 +6,7 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <TestGroupName>ProjectTemplates.Blazor.WebAssembly.Tests</TestGroupName>
     <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
+    <HelixTimeout>00:60:00</HelixTimeout>
```

```suggestion
    <HelixTimeout>01:00:00</HelixTimeout>
```

[`34da8e4` (#42548)](https://github.com/dotnet/aspnetcore/pull/42548/commits/34da8e43b8f72c47d21cb29716c2a3a4175ac51d)